qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -472,6 +472,9 @@ def _default_options(cls):
             accept_distributed_results=None,
             blocking_qubits=None,
             blocking_enable=False,
+            memory_blocking_bits=0,
+            batched_shots_optimization=True,
+            batched_shots_optimization_threshold=16,
```

Comment:
What does this mean, you need to add docs that explain what htis threshold value is.
If its num qubits  why so high? I thought the point of this to speed up low number of qubits?

Reply:
I added documents of batched_shots_optimization options in CONTRIBUTING.md 
batched shots optimization will be enabled when number of qubits of circuit is less than threshold. Because this technique does not improve performance for large number of qubits.

Reply:
I added documentation for batched_shots_optimization options in aer_simulator.py